src/search_options.hpp

```diff
@@ -495,6 +510,42 @@ void parseCommandLine(LambdaOptions & options, int argc, char const ** argv)
         options.geneticCodeQry = static_cast<seqan3::genetic_code>(geneticCodeTmp);
     }
 
+    if (options.profile == "fast")
+    {
+        if (options.nucleotide_mode)
+        {
+            options.iterativeSearch        = false;
```

Comment:
I thought we came to the conclusion that the iterativeSearch is always beneficial, or did we not find a good parameter-pair for nucleotide+fast?

Reply:
We did not find a good pair and you suggested to pick one of the search0 candidates as regular search for fast.